### PR TITLE
clamav: Update to version 0.101.4

### PR DESCRIPTION
**Pull request by BKPepe**
Maintainer: @ratkaj, @lucize
Compile tested: Turris Omnia, mvebu (cortex-a9), OpenWrt master
Run tested: Turris Omnia, mvebu (cortex-a9), OpenWrt master
Tests: clamd (foreground & background), clamdscan, clamscan

Description:

Update to version [0.101.4](https://blog.clamav.net/2019/08/clamav-01014-security-patch-release-has.html)

- Change deprecated options to a new one:
DetectBrokenExecutables to AlertBrokenExecutables
ArchiveBlockEncrypted to AlertEncrypted

- Fixes: [CVE-2019-12900](https://nvd.nist.gov/vuln/detail/CVE-2019-12900) and [CVE-2019-12625](https://security-tracker.debian.org/tracker/CVE-2019-12625)

### CODE REVIEW

**Comment by lucize** [comment]
LGTM, have you had a  chance to look into 0.102 ? especially libmilter binary

**Comment by BKPepe** [comment]
I wanted to update it to 0.102.0 in the `master` master, but I am dealing with some strange issues that it can not copy `clamav-milter` and as it is missing.

**Comment by lucize** [comment]
that's what I was talking about, it seems that the dev libs are not detected, something has changed in configure

File: net/clamav/files/clamav.init [hunk 1]
```diff
@@ -79,15 +79,15 @@ start_clamav_instance() {
 	echo "ScanPE " $ScanPE >> $CLAMD_CONFIGFILE
 	echo "DisableCertCheck " $DisableCertCheck >> $CLAMD_CONFIGFILE
 	echo "ScanELF " $ScanELF >> $CLAMD_CONFIGFILE
-	echo "DetectBrokenExecutables " $DetectBrokenExecutables >> $CLAMD_CONFIGFILE
```
Comment (neheb) [inline net/clamav/files/clamav.init]: This looks like it can be aggregated into a single >> call.

Comment (neheb) [reply net/clamav/files/clamav.init]: https://github.com/neheb/packages/commit/df0534bb702ec31ffcde027cb43e0d520bb12e08 is effectively what I meant.

I also meant single > call.

Comment (BKPepe) [reply net/clamav/files/clamav.init]: Will take a look at it today, thanks! I would like to test it before cherry-picking it.

Comment (BKPepe) [reply net/clamav/files/clamav.init]: Tested, thanks!

**Comment by neheb** [comment]
Great!

**Pull request merged**